src/docker-client.ts

```diff
@@ -27,7 +27,7 @@ export interface DockerClient {
   create(repoTag: RepoTag, env: Env, boundPorts: BoundPorts, cmd: Command[]): Promise<Container>;
   start(container: Container): Promise<void>;
   exec(container: Container, command: Command[]): Promise<ExecResult>;
-  buildImage(repoTag: RepoTag, context: BuildContext): Promise<void>;
+  buildImage(repoTag: RepoTag, context: BuildContext, buildParameters: {}): Promise<void>;
```

Comment:
`buildParameters: {}` is telling TypeScript that the type of `buildParameters` is an empty object, not an object. You probably wanted `buildParameters: object`, but it'd actually be better to define a custom type for this. Something like: `type BuildParameters = {[key: string]: string}`.

Reply:
I'll do a rework for the build arguments in a few minutes ;)